packages/shared/src/lib/featureValues.ts

```diff
@@ -23,4 +16,6 @@ export enum ScrollOnboardingVersion {
 export enum ExperimentWinner {
   ArticleOnboarding = 'v3',
   PostCardShareVersion = 'v2',
+  AuthVersion = 'v4',
+  OnboardingVersion = 'v2',
```

Comment:
What is this one?

Reply:
I just saw this along the way (a magic string) and so kept it in the right place. This is the adoption of the new onboarding (v2) and removal of legacy onboarding (v1).

Reply:
I see ok :)
Must have been an old one indeed